test/sqllogictest/transform/threshold_elision.slt

```diff
@@ -487,10 +487,10 @@ Explained Query:
           Threshold // { non_negative: true }
             Union // { non_negative: false }
               Distinct project=[#0, (#1 || "_iter")] // { non_negative: false }
-                Get l0 // { non_negative: false }
+                Get l0 // { non_negative: true }
               Negate // { non_negative: false }
                 Distinct project=[#0, (#1 || "_iter")] // { non_negative: false }
                   Filter (#0 > 1) // { non_negative: false }
-                    Get l0 // { non_negative: false }
+                    Get l0 // { non_negative: true }
 
```

Comment:
We probably want to chase down what this means in the context of the negative example. The `Get` registers as non-negative, but then the terms above it do not.